ci/docker_build_java.sh

```diff
@@ -25,10 +25,18 @@ mkdir -p /build/java
 
 arrow_src=/build/java/arrow
 
+# Remove any pre-existing artifacts
+rm -rf $arrow_src
+
 pushd /arrow
-  rsync -a header java format integration $arrow_src
+rsync -a header java format integration $arrow_src
 popd
 
 pushd $arrow_src/java
-  mvn -B -DskipTests -Drat.skip=true install
+mvn -B -DskipTests -Drat.skip=true install
```

Comment:
Just to confirm, this is build only and you don't want tests run?

Reply:
Ah, thanks for catching. Yeah we need to run the tests. I'll fix